pinc/Round.inc

```diff
@@ -565,19 +565,18 @@ function is_formatting_round($round)
  *
  * @param array $arr
  * @param string $key
- * @param mixed $default
+ * @param Round|null $default
  * @param bool $allownull
  *
+ * @return Round|null
+ *
```

Comment:
To follow up to myself. I see there _is_ in fact one other you might want PHPDocs: when PHPDocs type constraints can express something that type hints can't. The main examples of this are `array<string>`, which can also be written as `string[]`; and `array<k,v>` for associative arrays. None of these are supported by PHP at all yet. Another significant example is union types (eg `string|int`) which were added in PHP 8.0.

Reply:
Probably not? My instinct is to leave them in for now until we're on PHP 8.0 (for the union types and mixed) and we have a bit more critical mass of type defs in the functions.

Reply:
Sounds reasonable. PHPStan has warnings at the higher warning levels about un-annotated functions etc. I've disabled those warnings for now, but this will give us a path to gradually add them (on a per file basis) and to not backslide.